src/main/java/io/appium/java_client/android/AndroidMobileCommandHelper.java

```diff
@@ -291,4 +291,109 @@
+        Object[] values = new Object[]{phoneNumber, gsmCallActions.toString()};
+        return new AbstractMap.SimpleEntry<>(GSM_CALL, prepareArguments(parameters, values));
+    }
+
+    /**
+     * This method forms a {@link Map} of parameters for the element
+     * value replacement. It is used against input elements
+     *
+     * @param gsmSignalStrength One of available GSM signal strength
+     *
+     * @return a key-value pair. The key is the command name. The value is a {@link Map} command arguments.
+     */
+    public static Map.Entry<String, Map<String, ?>>  gsmSignalStrengthCommand(
+            GsmSignalStrength gsmSignalStrength) {
+        return new AbstractMap.SimpleEntry<>(GSM_SIGNAL,
+                prepareArguments("signalStrengh", gsmSignalStrength.getValue()));
+    }
+
+    /**
+     * This method forms a {@link Map} of parameters for the element
+     * value replacement. It is used against input elements
+     *
+     * @param gsmVoiceState One of available GSM voice state
+     *
+     * @return a key-value pair. The key is the command name. The value is a {@link Map} command arguments.
+     */
+    public static Map.Entry<String, Map<String, ?>>  gsmVoiceCommand(
+            GsmVoiceState gsmVoiceState) {
+        return new AbstractMap.SimpleEntry<>(GSM_VOICE,
+                prepareArguments("state", gsmVoiceState.toString()));
+    }
+
+    /**
+     * This method forms a {@link Map} of parameters for the element
+     * value replacement. It is used against input elements
+     *
+     * @param networkSpeed One of possible NETWORK_SPEED values
+     *
+     * @return a key-value pair. The key is the command name. The value is a {@link Map} command arguments.
+     */
+    public static Map.Entry<String, Map<String, ?>>  networkSpeedCommand(
+            NetworkSpeed networkSpeed) {
+        return new AbstractMap.SimpleEntry<>(NETWORK_SPEED,
+                prepareArguments("netspeed", networkSpeed.toString()));
+    }
+
+    /**
+     * This method forms a {@link Map} of parameters for the element
+     * value replacement. It is used against input elements
+     *
+     * @param percent A number in range [0, 4]
+     *
+     * @return a key-value pair. The key is the command name. The value is a {@link Map} command arguments.
+     */
+    public static Map.Entry<String, Map<String, ?>>  powerCapacityCommand(
+            int percent) {
+        return new AbstractMap.SimpleEntry<>(POWER_CAPACITY,
+                prepareArguments("percent", percent));
+    }
+
+    /**
+     * This method forms a {@link Map} of parameters for the element
+     * value replacement. It is used against input elements
+     *
+     * @param powerACState One of available power AC state
+     *
+     * @return a key-value pair. The key is the command name. The value is a {@link Map} command arguments.
+     */
+    public static Map.Entry<String, Map<String, ?>>  powerACCommand(
```

Comment:
all these methods ahev one extra space between the return type and the method name